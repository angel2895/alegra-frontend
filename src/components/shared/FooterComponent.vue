<template>
  <footer>Â© Angel Orellana</footer>
</template>

<script lang="ts">
import { Vue } from "vue-class-component";

export default class FooterComponent extends Vue {}
</script>

<style>
footer {
  height: 100px;

  display: flex;
  justify-content: center;
  align-items: center;

  font-size: 1.4rem;
  font-weight: bold;
  color: white;

  background-color: var(--primary-color);
}
</style>
