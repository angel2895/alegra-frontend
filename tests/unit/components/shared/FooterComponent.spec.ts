import { shallowMount } from "@vue/test-utils";

import FooterComponent from "@/components/shared/FooterComponent.vue";

describe("Footer Component", (): void => {
  it("Match with the snapshot", () => {
    const wrapper = shallowMount(FooterComponent);
    expect(wrapper.html()).toMatchSnapshot();
  });

  it("Name (Angel Orellana) in the footer ", (): void => {
    const wrapper = shallowMount(FooterComponent);
    const $footer = wrapper.find("footer");
    expect($footer.text()).toBe("Â© Angel Orellana");
  });
});
