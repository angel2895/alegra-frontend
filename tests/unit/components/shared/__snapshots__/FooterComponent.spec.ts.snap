// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Component Match with the snapshot 1`] = `<footer>Â© Angel Orellana</footer>`;
